modules/custom/social_advanced_image/social_advanced_image.services.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  plugin.manager.image_copyright_attributes:
+    class: Drupal\social_advanced_image\AdvancedImageAttributesManager
+    arguments: ['@module_handler', '@cache.discovery']
+
+  image_copyright.overrider:
```

Comment:
As part of the renaming effort, don't forget to rename this to something with `advanced_image` instead of `image_copyright`.